src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and should not contain any "
```

Comment:
I do agree that the previous implementation has the contraints within each class. However, it has mostly been refactored to the messagePackage, perhaps this should be in the contraintMessage class